docs/pipelines/tasks/_shared/yaml/PublishTestResultsV2.md

```diff
@@ -2,8 +2,8 @@
 # Publish Test Results to Azure Pipelines/TFS
 - task: PublishTestResults@2
   inputs:
-    #testResultsFormat: 'JUnit' # Options: JUnit, NUnit, VSTest, xUnit
-    #testResultsFiles: '**/TEST-*.xml' 
+    #testRunner: 'JUnit' # Options: JUnit, NUnit, VSTest, xUnit

```

Comment:
[An alias](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/PublishTestResultsV2/task.json#L32) named `testResultsFormat` was intentionally created to be used instead of `testRunner` because often it isn't JUnit that runs the tests (for example), but JUnit is the results format.